content/_data/changelogs/weekly.yml

```diff
@@ -2235,6 +2235,79 @@
       # Also Chinese i18n but too minor otherwise
     # pull: 3288 too minor (French i18n fix)
     # pull: 3322 too minor (more Chinese i18n)
+- version: "2.111"
+  date: 2018-03-11
+  changes:
+    - type: rfe
+      issue: 46652
+      pull: 3254
+      message: >
+        Pipeline builds could not be started if the Authorize Project plugin was configured to associate the build with a user to whom the authorization strategy was configured to deny Agent/Build permission on the master node.
+    - type: rfe
+      issue: 50056
+      pull: 3228
+      message: >
+        Reduce memory footprint of <code>jenkins.model.lazy.AbstractLazyLoadRunMap#search</code> in descending order.
+    - type: rfe
+      pull: 3323
+      message: >
+        Improve Dutch translation.
```

Comment:
We usually put major translations in the bottom and skip small ones like this one

Reply:
Right, I added the next RFE later, so this isn't the last one.

Regarding size, it looked like more than it is from the number of files, I admit I didn't check it in more detail. I'll make this a comment.